docs/versioned/admin/admin-overview.md

```diff
@@ -0,0 +1,125 @@
+---
+audience: administrator
+components:
+  - serving
+  - eventing
+function: reference
+---
+# Overview
+
+Knative has a set of tools and capabilities to administer your Kubernetes clusters. This article provides a overview of Knative features, capabilities, and resources of interest to Kubernetes Administrators, and is organized by the following areas:
+
+- Installation
+- Configuration
+- Monitoring and Observability
+- Security and Access Control
+- Updates and Maintenance
+
+## Installation
+
+Use the Installation Roadmap for guidance on installing Knative using the resources and tool choices that best fit your needs. You can install using YAML resources, manifests, or CLI tools that include the Knative Operator.
+
+The Knative Operator is a custom controller that extends the Kubernetes API to install Knative components.
+
+You can install Knative components in three ways:
+
+- Use a [YAML-based installation](/install/yaml-install/README.md)
+- Use the [Knative Operator CLI plugin](/install/operator/knative-with-operator-cli.md)
+- Use the [Knative Operator](/install/operator/knative-with-operators.md) with YAML resource files and the `kubectl` CLI.
+
+    - [Knative Serving installation files](/install/yaml-install/serving/serving-installation-files.md)
+    - [Knative Eventing installation files](/install/yaml-install/eventing/eventing-installation-files.md)
+
+### Recommended plugins
+
+You can also install these plugins service to extend Knative capabilities for service meshes and application security:
+
+- [Istio for Knative](/install/installing-istio.md)
+- [Knative Backstage plugin](/install/installing-backstage-plugins.md)
+- [Installing Security-Guard](/serving/app-security/security-guard-install.md)
```

Comment:
Security Guard is a bit of an experiment; I wouldn't necessarily recommend it as part of a core installation.